deploy/rbd/kubernetes/csi-rbdplugin.yaml

```diff
@@ -50,14 +50,19 @@ spec:
             - "--endpoint=$(CSI_ENDPOINT)"
             - "--v=5"
             - "--drivername=csi-rbdplugin"
-            - "--containerized=true"            
+            - "--containerized=true"
+            - "--metadatastorage=node"
```

Comment:
IMHO k8s deployments should default to `--metadatastorage=k8s_configmap` @rootfs ?

Reply:
yes, please make that change @mickymiek 
btw, sorry for late response.

Reply:
@rootfs done